projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -149,3 +154,53 @@ export function verifyTicketDoesNotExist(ticketDetails: TestTicketDetails) {
 export function visitApparelUKTicketListingPage(){
   visitPage('apparel-uk-spa/en/GBP/my-account/support-tickets', 'apparelTicketListingPage');
 }
+
+export function verifyTicketListingTableContent(): number{
```

Comment:
This function is doing two things. please split it into two.
Maybe:  GetNumberOfTicketsListed and verifyTicketListingTableContents

Reply:
No change here?